src/main/webapp/js/common.js

```diff
@@ -484,20 +484,23 @@ function getPointValue(s, ditchZero) {
 /**
  * Checks if element is within browser's viewport.
  * @return true if it is within the viewport, false otherwise
+ * @see http://stackoverflow.com/q/123999
  */
 function isWithinView(element) {
-    var viewHeight = window.innerHeight;
-    var viewTop = window.scrollY;
-    var viewBottom = viewTop + viewHeight;
+    var baseElement = $(element)[0]; // unwrap jquery element
+    var rect = baseElement.getBoundingClientRect();
     
-    var elementHeight = element.offsetHeight;
-    var elementTop = element.offsetTop;
-    var elementBottom = elementTop + elementHeight;
+    var $viewport = $(window);
     
-    return viewHeight >= elementHeight
-           ? viewTop <= elementTop && viewBottom >= elementBottom          // all within view
-           : viewTop <= elementTop && viewBottom >= elementTop             // top within view
-             || viewTop <= elementBottom && viewBottom >= elementBottom;   // btm within view
+    // subpixel range to prevent rounding errors
+    // e.g. rect.top = -0.0013 will still be considered to be within viewport
```

Comment:
Should be a `/** ... */` comments instead?
Not clear why you used `0.0013` in the comment. 
I guess the tolerance is used to consider a rectangle that is slightly _bigger_ than the specified one. Is that the case? If so, may be explain it like that?
The unexplained term `subpixel range` doesn't seem to be necessary to the explanation.